### ISSUE TITLE
Playground-Win32 crashes because of DispatcherQueue

### ISSUE THREAD

**Issue by asklar** [microsoft/react-native-windows#4833]
Load rntester or any page in playground Win32 (XAML island), get a crash here:

```
React.UWP.dll!winrt::throw_hresult(const   winrt::hresult result) Line 4673
--
React.UWP.dll!winrt::check_hresult(const   winrt::hresult result) Line 4735
React.UWP.dll!winrt::impl::consume_Windows_ApplicationModel_Core_ICoreImmersiveApplication<winrt::Windows::ApplicationModel::Core::ICoreImmersiveApplication>::MainView()   Line 347
React.UWP.dll!winrt::Windows::ApplicationModel::Core::CoreApplication::MainView::__l2::<lambda>(const   winrt::Windows::ApplicationModel::Core::ICoreImmersiveApplication & f)   Line 1173
React.UWP.dll!winrt::Windows::ApplicationModel::Core::CoreApplicationView   <lambda>(const   winrt::Windows::ApplicationModel::Core::ICoreImmersiveApplication   &)::<lambda_invoker_stdcall>(const   winrt::Windows::ApplicationModel::Core::ICoreImmersiveApplication & f)   Line 1173
React.UWP.dll!winrt::impl::factory_cache_entry<winrt::Windows::ApplicationModel::Core::CoreApplication,winrt::Windows::ApplicationModel::Core::ICoreImmersiveApplication>::call<winrt::Windows::ApplicationModel::Core::CoreApplicationView   (__stdcall*)(winrt::Windows::ApplicationModel::Core::ICoreImmersiveApplication   const   &)>(winrt::Windows::ApplicationModel::Core::CoreApplicationView(__stdcall*)(const   winrt::Windows::ApplicationModel::Core::ICoreImmersiveApplication &)   && callback) Line 5984
React.UWP.dll!winrt::impl::call_factory_cast<winrt::Windows::ApplicationModel::Core::CoreApplicationView   (__stdcall*)(winrt::Windows::ApplicationModel::Core::ICoreImmersiveApplication   const &),winrt::Windows::ApplicationModel::Core::CoreApplication,winrt::Windows::ApplicationModel::Core::ICoreImmersiveApplication,winrt::Windows::ApplicationModel::Core::CoreApplicationView   <lambda>(const   winrt::Windows::ApplicationModel::Core::ICoreImmersiveApplication   &)>(winrt::Windows::ApplicationModel::Core::CoreApplication::MainView::__l2::winrt::Windows::ApplicationModel::Core::CoreApplicationView   <lambda>(const   winrt::Windows::ApplicationModel::Core::ICoreImmersiveApplication &)   && callback) Line 6023
React.UWP.dll!winrt::Windows::ApplicationModel::Core::CoreApplication::MainView()   Line 1173
React.UWP.dll!Mso::DispatchQueueStatic::MakeMainUIScheduler()   Line 252
React.UWP.dll!Mso::DispatchQueueStatic::MainUIQueue()   Line 273
React.UWP.dll!Mso::DispatchQueue::MainUIQueue()   Line 549
React.UWP.dll!react::uwp::AppearanceChangeListener::AppearanceChangeListener(std::weak_ptr<react::uwp::IReactInstance>   && reactInstance) Line 18
React.UWP.dll!Mso::Internal::ActiveObjectMakePolicy::Make<react::uwp::AppearanceChangeListener,Mso::WeakRefCountMemoryGuard<react::uwp::AppearanceChangeListener,Mso::ObjectWeakRefContainer<react::uwp::AppearanceChangeListener,Mso::ObjectWeakRef>>,std::shared_ptr<react::uwp::IReactInstance>   const   &>(Mso::WeakRefCountMemoryGuard<react::uwp::AppearanceChangeListener,Mso::ObjectWeakRefContainer<react::uwp::AppearanceChangeListener,Mso::ObjectWeakRef>>   & memoryGuard, const std::shared_ptr<react::uwp::IReactInstance>   & <args_0>) Line 169
React.UWP.dll!Mso::Make<react::uwp::AppearanceChangeListener,react::uwp::AppearanceChangeListener,std::shared_ptr<react::uwp::IReactInstance>   const &>(const std::shared_ptr<react::uwp::IReactInstance> &   <args_0>) Line 28
React.UWP.dll!react::uwp::UwpReactInstance::Start(const   std::shared_ptr<react::uwp::IReactInstance> & spThis, const   react::uwp::ReactInstanceSettings & settings) Line 118
Playground-Win32.exe!WindowData::OnCommand(HWND__   * hwnd, int id, HWND__ * hwndCtl, unsigned int __formal) Line 150
Playground-Win32.exe!WndProc(HWND__ *   hwnd, unsigned int message, unsigned int wparam, long lparam) Line 394
```

@vmoroz is aware of the issue

**Comment by NickGerleman** [comment]
Should be fixed by #4877. Will be backported to 0.62.

**Comment by asklar** [comment]
@NickGerleman please do not close issue out from under triage. Thanks.
